src/signaling/backend_configuration.go

```diff
@@ -70,40 +72,15 @@ func NewBackendConfiguration(config *goconf.ConfigFile) (*BackendConfiguration,
 			compat: true,
 		}
 	} else if backendIds, _ := config.GetString("backend", "backends"); backendIds != "" {
-		seenIds := make(map[string]bool)
-		for _, id := range strings.Split(backendIds, ",") {
-			id = strings.TrimSpace(id)
-			if id == "" {
-				continue
+		for host, backendIDs := range getConfiguredHosts(config) {
+			for id, configuredBackends := range backendIDs {
+				backends[host] = append(backends[host], configuredBackends...)
+				var u strings.Builder
```

Comment:
With the map structure change below you can use the (single) backend directly, no need for `strings.Builder`.